app/components/views/GetStartedPage/PreCreateWallet/LedgerLoaderBarContainer/LedgerLoaderBarContainer.jsx

```diff
@@ -0,0 +1,33 @@
+import { FormattedMessage as T } from "react-intl";
+import styles from "./LedgerLoaderBarContainer.module.css";
+import { useLedgerLoaderBarContainer } from "./hooks";
+
+const LedgerLoaderBarContainer = ({ loaderBar }) => {
+  const { ledgerDevice, deviceLabel } = useLedgerLoaderBarContainer();
+  return ledgerDevice ? (
```

Comment:
```suggestion
  return ledgerDevice && (
```

Reply:
changed

Reply:
Is it right to change this? Looks like it does the other thing if no label.